src/test/architecture/test_architecture_loader.py

```diff
@@ -870,9 +956,15 @@ def test_callback_name(self, mocker):
             callback_id[2], node.node_name, node.node_id, symbol[2], topic_name, None)
         sub_cb_1 = SubscriptionCallbackValue(
             callback_id[3], node.node_name, node.node_id, symbol[3], topic_name, None)
+        srv_cb_0 = SubscriptionCallbackValue(
+            callback_id[4], node.node_name, node.node_id, symbol[4], service_name, None)
+        srv_cb_1 = SubscriptionCallbackValue(
+            callback_id[5], node.node_name, node.node_id, symbol[5], service_name, None)
```

Comment:
```suggestion
        srv_cb_0 = ServiceCallbackValue(
            callback_id[4], node.node_name, node.node_id, symbol[4], service_name, None)
        srv_cb_1 = ServiceCallbackValue(
            callback_id[5], node.node_name, node.node_id, symbol[5], service_name, None)
```

Reply:
addressed in : [7cf142b](https://github.com/tier4/CARET_analyze/pull/192/commits/7cf142b6b6696d3a2787d716ae4ee7ea60f233cd)

Reply:
addressed in [1d2dc97](https://github.com/tier4/CARET_analyze/pull/192/commits/1d2dc976a3a5b06677e5eacb5d55d80db02364ec)